cirq-core/cirq/value/condition.py

```diff
@@ -135,6 +136,107 @@ def _qasm_(self, args: 'cirq.QasmArgs', **kwargs) -> Optional[str]:
         return f'{key}==1'
 
 
+@attrs.frozen
+class BitMaskKeyCondition(Condition):
```

Comment:
Curious why create a new class rather than add the functionality to KeyCondition?

Reply:
I prefer to leave KeyCondition as a simple condition and not overload it